packages/notebook-extension/schema/tracker.json

```diff
@@ -377,11 +377,17 @@
       "type": "boolean",
       "default": true
     },
-    "nonObservedBottomMargin": {
-      "title": "Non-observed bottom margin",
-      "description": "Defines the non-observed bottom margin for the virtual notebook, set a positive number of pixels to render cells below the visible view",
+    "observedTopMargin": {
+      "title": "Observed top margin",
+      "description": "Defines the observed top margin for the virtual notebook, set a positive number of pixels to render cells above the visible view",
       "type": "string",
-      "default": "0px"
+      "default": "1000px"
```

Comment:
Probably not worth for this PR. Would taking into account zoom levels and have an "auto" option that sets this size taking into account the current screen height, for example?

Reply:
Great call, Yeah, I was thinking to define a margin with the CSS vh unit measure (https://www.w3schools.com/cssref/css_units.asp)

As I had not time to test that and I have seen good results with `1000px` which is a safety default, I was thinking to try it after.

Reply:
vh will compute the size of the viewport, but the notebook constructor would need to get that size. That value is used in the notebook constructor and we need to make sure that we have access to the notebook panel viewport size at that place. Definitively something to try in a next iteration.

Reply:
Agreed. This makes sense for the next iteration. Not a blocker for this one.